packages/launcher/lib/detect.ts

```diff
@@ -15,7 +15,14 @@ import {
 } from './types'
 import * as windowsHelper from './windows'
 
-const setMajorVersion = (browser: FoundBrowser) => {
+type HasVersion = {
+  version?: string
+  majorVersion?: string | number
+  name: string
+}
```

Comment:
```ts
type HasVersion = Pick<FoundBrowser, 'version' | 'majorVersion' | 'name'>
```

Does our typescript version support `Pick`? Would be easier to maintain

Reply:
I tried using Pick - but it messed up the return type :(